src/main/java/wolox/training/models/Book.java

```diff
@@ -51,6 +53,10 @@
     @Column(nullable = false, unique =true)
     private String isbn;
 
+    @ManyToOne
```

Comment:
User is your most important or strongest class, since it is the one that has the methods in charge of adding or removing elements to the collection.
Book all you have to do is map your collection to the user collection. With this, every time the user collections are modified, the book collection will be affected.

```suggestion
    @ManyToMany(mappedBy = "books")
    private List <User> users = new ArrayList <> ();
```

Reply:
I changed  the annotation.